L/libdeflate/build_tarballs.jl

```diff
@@ -3,20 +3,36 @@
 using BinaryBuilder, Pkg
 
 name = "libdeflate"
-version_string = "1.13"
-version = VersionNumber(version_string)
+version = v"1.18"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://github.com/ebiggers/libdeflate/archive/refs/tags/v$(version_string).tar.gz", 
-                  "0d81f197dc31dc4ef7b6198fde570f4e8653c77f4698fcb2163d820a9607c838"),
+    GitSource(
+        "https://github.com/ebiggers/libdeflate",
+        "495fee110ebb48a5eb63b75fd67e42b2955871e2"
+    ),
+    ArchiveSource(
+        "https://github.com/phracker/MacOSX-SDKs/releases/download/10.15/MacOSX10.14.sdk.tar.xz",
+        "0f03869f72df8705b832910517b47dd5b79eb4e160512602f593ed243b28715f"
+    ),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/libdeflate-*/
-make PROG_SUFFIX=$exeext PREFIX=${prefix} LIBDIR=${libdir} DISABLE_ZLIB=true
-make PROG_SUFFIX=$exeext PREFIX=${prefix} LIBDIR=${libdir} install
+# This requires macOS 10.14
```

Comment:
On slack you said you require 10.13, not 10.14

Reply:
Right - I forgot to change it to 10.13 after it worked! Thank you for the help